.github/workflows/p2p.yaml

```diff
@@ -0,0 +1,80 @@
+name: P2P
```

Comment:
Should we call this, and its file name idp-p2p? I am just thinking about future proofing us providing a CECG P2P that doesn't do any IDP stuff e.g. the login etc